code/online_challenges/src/Hacker_Rank/3D_Surface_Area/readme

```diff
@@ -0,0 +1 @@
+Problem Link: https://www.hackerrank.com/challenges/3d-surface-area/problem?h_r=internal-search
```

Comment:
Please include more than just a link. A markdown file with problem/solution details (like for other online challenges) would be best.